packages/banner/.stylelintrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Possible to not use from the root package stylelint?

Reply:
Will do @abhilashlr89